gcp/extract/lib/results.py

```diff
@@ -139,12 +139,16 @@ def sum_into_data(root, basenames, columns, config, transforms, vectransforms):
             except:
                 print "Failed to read " + os.path.join(targetdir, basenames[ii] + '.nc4')
                 traceback.print_exc()
-
+            last_targetdir = targetdir
     print "Observations:", observations
     if observations == 0:
         print message_on_none
+    
+    do_deltamethod = False if configs.is_parallel_deltamethod(config) else config.get('deltamethod', None)
+    filepath = os.path.join(last_targetdir, basenames[0] + '.nc4')
+    years, regions, data_not = bundles.read(filepath = filepath, column = columns[0], deltamethod = do_deltamethod)
```

Comment:
I don't think we need this. Either the `try` clause above successfully ran, in which case `years` is already defined; or it didn't, in which case this would produce an error. I suggest putting `years = []` at the top of this function, removing these lines, and just using whatever the `years` variable ends up being set to.

Reply:
Also drop the `last_targetdir` line above, if you go this route.

Reply:
makes a lot more sense 💯

Reply:
so, the logic here is that in python variables still exist from the loop they were generated in after the loop ends. I can google this for more info. just learning out loud in case this is obviously wrong.